mantidproject/mantid
# Convert Group1 from Python 2 to Python 3

This PR deals with the Python2 to Python3 conversion of some Utility-related files in Group 1 of #18550

The files are related to the Interfaces in the folder `Utility`

## To test:

Code review

1. Go to Interfaces > Utility
2. Open each of the interfaces `FitlerEvents`, `QECoverage` and `TofConverter`
  * Confirm that there are no import or general errors when opening the interfaces
3. Play around with the interfaces (no data needed)
  * Confirm that there are no Python related errors

Fixes #18805

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
